frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboard.tsx

```diff
@@ -223,6 +224,8 @@ class PublicOrEmbeddedDashboardInner extends Component<PublicOrEmbeddedDashboard
       dashcard => !isActionDashCard(dashcard),
     );
 
+    const downloadsEnabled = PLUGIN_DOWNLOADS_ENABLED.areDownloadsEnabled();
```

Comment:
This is where I'll pass down the hash parameters (hide_download_button and downloads) in the next task